test/no-focused-test.spec.js

```diff
@@ -0,0 +1,63 @@
+const { RuleTester } = require('eslint');
+const rule = require('../lib/rules/no-focused-test');
+
+RuleTester.setDefaultConfig({
+  parserOptions: {
+    ecmaVersion: 2018,
+  },
+});
+
+const invalid = (code, output) => ({
+  code,
+  errors: [
+    {
+      messageId: 'noFocusedTest',
+      suggestions: [{ messageId: 'removeFocusedTestAnnotation', output }],
+    },
+  ],
+});
+
+const valid = (code) => ({
+  code,
+});
+
```

Comment:
If the valid cases don't need any wrapping, they can just be strings

```suggestion
```

Reply:
Thanks!)